src/settings/LoginSettingsViewer.ts

```diff
@@ -127,13 +127,19 @@ export class LoginSettingsViewer implements UpdatableSettingsViewer {
 			],
 			selectedValue: logins.getUserController().userSettingsGroupRoot.usageDataOptedIn,
 			selectionChangedHandler: v => {
-				logins.getUserController().userSettingsGroupRoot.usageDataOptedIn = v
-				locator.entityClient.update(logins.getUserController().userSettingsGroupRoot)
+				if (isNotNull(v)) {
```

Comment:
how can it be `null`?

Reply:
![grafik](https://user-images.githubusercontent.com/97233129/199732446-3f609d62-2018-41e6-a12a-0a153f40c896.png)
It complains if I don't add that.

Reply:
I can add an assertNotNull instead since it's not selectable

Reply:
I didn't see that there's a possible `null` for undecided. It should not be nullable here tho so yes, assert pls